doc/htmldoc/index.rst

```diff
@@ -75,6 +83,36 @@ Welcome to the NEST Simulator documentation
 
              Discover all our examples!
 
+.. grid::
+
+    .. grid-item::
+        :columns: 6
+        :child-direction: row
+        :child-align: end
+
+        .. raw:: html
+
+           <object width="89%" height="89%" data="_static/img/excalidraw_nestconcept.svg" type="image/svg+xml"></object>
+
+
+    .. grid-item::
+        :columns: 5
+
+        **How NEST works - Conceptual approach**
+
+        This diagram provides an overview of the components essential for building and simulating a network model with NEST:
+
+        A NEST simulation is created with input from :doc:`stimulation devices </models/index_generator>`,
+        :doc:`neuron models </models/index_neuron>`, and :doc:`synapse models </models/index_synapse>`,
+        along with  :ref:`connection rules <connectivity_concepts>`.
+        You can choose what data to record with :doc:`recording devices </models/index_recorder>`.
+        After simulation, the output is ready for analysis with external tools.
```

Comment:
We should also point to our built-in analysis support and advertise our friends a little. I can never remember how to link in RST, I hope you can fix my sketch below. 
```suggestion
        After simulation, the output is ready for analysis with NEST's built in <raster_plot> and <voltage_trace> modules or external tools such as <Elephant>.
```